homeassistant/components/template/config.py

```diff
@@ -39,6 +58,7 @@
         vol.Optional(CONF_TRIGGER): cv.TRIGGER_SCHEMA,
         vol.Optional(CONF_CONDITION): cv.CONDITIONS_SCHEMA,
         vol.Optional(CONF_ACTION): cv.SCRIPT_SCHEMA,
+        vol.Optional(CONF_VARIABLES): cv.SCRIPT_VARIABLES_SCHEMA,
```

Comment:
Don't forget to document this since it will be available for all template entities and not only blueprint templates.

Reply:
https://github.com/tetele/home-assistant.io/blob/8f0d137550888bc76fb12d6f28aa6df4a3321e2d/source/_integrations/template.markdown?plain=1#L146-L154